pkg/cli/mirror/create.go

```diff
@@ -144,7 +144,7 @@ func (o *MirrorOptions) run(ctx context.Context, cfg *v1alpha2.ImageSetConfigura
 	}
 
 	if len(cfg.Mirror.Samples) != 0 {
-		klog.V(4).Info("sample images full not implemented")
+		klog.Info("sample images full not implemented")
```

Comment:
Not sure why we move away from verbosity setting for this logging? Is this intentional?